components/style/properties/data.py

```diff
@@ -615,9 +642,14 @@ def _add_logical_props(data, props):
             props.add(prop.name)
 
 
+def remove(set_, item):
+    if item in set_:
```

Comment:
I was intentionally using `remove()` so that getting it wrong errored loud, fwiw...

Reply:
This is intended to catch a typo in the name, right?

My reasoning was that I don’t want to conditionally remove based on the engine, because we’d forget to edit this back when adding that property to the new layout engine.

I also tried not having a property exist at all before it is implemented in layout, but that’s already a lost cause: various parts of the style and script crate already refer to Rust APIs for specific APIs, so I used a pref to disable parsing instead. I’ll do the same here.